example-mojaloop-backend/values.yaml

```diff
@@ -326,48 +197,25 @@ centralledger-obj:
   ## Enable persistence using Persistent Volume Claims
   ## ref: https://kubernetes.io/docs/user-guide/persistent-volumes/
   ##
-  persistence:
-    ## @param persistence.enabled Enable MongoDB(&reg;) data persistence using PVC
-    ##
-    enabled: false
+  # persistence:
+  #   ## @param persistence.enabled Enable MongoDB(&reg;) data persistence using PVC
+  #   ##
+  #   enabled: false
 
 ## TODO: Add 
 cep-mongodb:
   enabled: true
 
-  ## @param nameOverride String to partially override mongodb.fullname template (will maintain the release name)
+  ## @param fullnameOverride String to fully override mongodb.fullname template (will maintain the release name)
   ##
-  nameOverride: cep-mongodb
-
+  fullnameOverride: cep-mongodb
 
   ## @param affinity Affinity for pod assignment
   ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
   ## Note: podAffinityPreset, podAntiAffinityPreset, and  nodeAffinityPreset will be ignored when it's set
   ##
   affinity: {}
 
-  ## @param nodeSelector Node labels for pod assignment
```

Comment:
Why are these removed?

Reply:
again they are the default in the bitnami chart so they are not needed here